src/debug-bar.js

```diff
@@ -62,13 +72,12 @@ function DebugBar() {
                 {fixture.title}
               </button>
             ))}
+            <button className="debug-bar-toggle-button" type="button" onClick={() => setShow(!show)}>
```

Comment:
This button will only be visible when show is true, so the debug bar cannot be opened again.
Maybe add
{!show
  <button className="debug-bar-toggle-button" type="button" onClick={() => setShow(true)}>
    Debug
  </button>
}
below the {show && } code